pkg/util/kubernetes/hpa/datadogexternal.go

```diff
@@ -24,14 +24,28 @@ var (
 	ddRequests = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "datadog_requests",
-			Help: "Counter of requests made to datadog",
+			Help: "Counter of requests made to Datadog",
 		},
 		[]string{"status"},
 	)
+	metricsEval = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "external_metrics_processed_value",
+		Help: "value processed from querying Datadog",
+	},
+		[]string{"metric"},
+	)
+	metricsPrecision = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Could we rename the metric to `metricsDelay` to be more accurate?
I would expect precision to refer to the value itself, not the timestamp (same at line 133)